base/applications/utilman/lang/pt-PT.rc

```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:         ReactOS Utility Manager (Accessibility)
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         English (United States of America) translation resource
```

Comment:
Purpose "english is wrong". I recommend to strip that entire comment header in pt-PT.rc instead of duplicating the content.